ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java

```diff
@@ -55,6 +58,7 @@ public void addUserTest() throws Exception {
             assertTrue(manageUserPage.getUserSavedNotification().contains("User Saved"));
 
             homePage = new HomePage(goToLoginPage().login("super_nurse", "Nurse123"));
+            homePage.waitForPage();
```

Comment:
Though the text to be checked by the function `getLoggedUsername() `is loaded in `DOM` after the login operation But it takes longer to be rendered on the `UI` which makes it to be invisible during the action `homePage.getLoggedUsername().contains("super_nurse");`.